pegasus/test/test_asset_helpers.rb

```diff
@@ -2,26 +2,100 @@
 require_relative '../../lib/cdo/pegasus'
 require 'minitest/autorun'
 
-CDO.dashboard_assets_dir = './test/fixtures/dashboard_assets'
+# AssetMap must be re-initialized before each test with CDO.dashboard_assets_dir set properly
```

Comment:
I've removed this comment. I was trying to explain that the singleton needs to be initialized before each test run, however there's no need for this warning since it will be impossible for someone to ignore this without breaking the tests.